package.json

```diff
@@ -179,6 +179,11 @@
         "command": "java.view.package.renameFile",
         "title": "%contributes.commands.java.view.package.renameFile%",
         "category": "Java"
+      },
+      {
+        "command": "java.updateDeprecatedTask",
```

Comment:
Will the command work if we do not register it in `package.json`? I guess it should be.

Reply:
It works, let's remove the command registration.